chrome/script.browserify.js

```diff
@@ -49,11 +49,18 @@ function view() {
         };
 
         var store = "default";
+        var storeTitle = "Default password store";
         var name = login;
         var i;
         if ((i = login.indexOf(":"))) {
           if (searchSettings && searchSettings.customStores.length > 1) {
             store = login.substr(0, i);
+            searchSettings.customStores.map(function(customStore) {
```

Comment:
unusual use of `map` 🙂

Reply:
Oops - that was a mistake. Intention was `forEach`. I think I must have been looking at `map` elsewhere, and typed it accidentally.

No real functional impact using it here though - so we can leave it as-is - but would you like me to change it for clarity?

Reply:
Or I guess I could have used a `(for i in searchSettings.customStores) {` loop too.

Reply:
I'll maybe change to `forEach` next time I need to do some changes in that file, as you said there is no functional impact anyway 😉